app/data_client.go

```diff
@@ -109,6 +109,37 @@ type TabularData struct {
 	TimeReceived  time.Time
 }
 
+// ExportTabularDataReturn represents the result of an ExportTabularData API call.
+type ExportTabularDataReturn struct {
+	OrganizationID   string
+	LocationID       string
+	RobotID          string
+	RobotName        string
+	PartID           string
+	PartName         string
+	ComponentName    string
+	ComponentType    string
+	MethodName       string
+	TimeCaptured     *timestamppb.Timestamp
+	MethodParameters map[string]*anypb.Any
```

Comment:
Similarly, `*anypb.Any` should use an `Any` type that's defined in the app package. I actually have a PR up that's in review right now that introduces this `Any` type, so you can use this type as well.

Reply:
If you merge main into this PR, you'll see a new `GenericProtoMessage` type in `common.go` that you can use!

Reply:
Added!